contracts/implementation/PoolCommitter.sol

```diff
@@ -438,6 +438,7 @@ contract PoolCommitter is IPoolCommitter, Initializable {
                 // Another update interval has passed, so we have to do the nextIntervalCommit as well
                 burnFeeHistory[updateIntervalId] = burningFee;
                 executeGivenCommitments(totalPoolCommitments[updateIntervalId]);
+                emit ExecutedCommitsForInterval(updateIntervalId);
```

Comment:
Add a `dev` entry to the doc comment for this function mentioning that this gets emitted.

Reply:
https://github.com/tracer-protocol/perpetual-pools-contracts/pull/296/commits/bf1df55b510595cdb4ff67bfc0e72ede05757753